mgrachev/3Sum/Program.cs

```diff
@@ -0,0 +1,65 @@
+﻿/*
+3Sum
+
+Given an integer array nums, return all the triplets [nums[i], nums[j], nums[k]] such that i != j, i != k, and j != k, and nums[i] + nums[j] + nums[k] == 0.
+
+Notice that the solution set must not contain duplicate triplets.
+
+------------
+
+Solution:
+
+Brute-force solution is to operate three pointers (three indices) and thus perform 3 loop iterations for the given array.
```

Comment:
As always good description. :)

Reply:
Thanks, @PShchegolevatykh I appreciate it! Will do my best to keep it up!